server/chat-commands/core.ts

```diff
@@ -349,29 +348,30 @@ exports.commands = {
 	authlist: 'authority',
 	authority(target, room, user, connection) {
 		if (target) {
-			let targetRoom = Rooms.search(target);
-			let availableRoom = targetRoom && targetRoom.checkModjoin(user);
+			const targetRoom = Rooms.search(target);
+			const availableRoom = targetRoom?.checkModjoin(user);
 			if (targetRoom && availableRoom) return this.parse(`/roomauth1 ${target}`);
 			return this.parse(`/userauth ${target}`);
 		}
 		/** @type {{[k: string]: string[]}} */
```

Comment:
```suggestion
```

Reply:
oops, i can't see apparently - will batch this in a sec